### ISSUE TITLE
How to display the current file ?

### ISSUE THREAD

**Issue by Ducatel** [scniro/gulp-clean-css#2]
Hi,

I don't find how display the current file name/path it will process in my gulp process.
I use this code but the arguments "details" doesn't contains the current file.
```
cleanCSS({debug: true}, function(details) {
    console.log("CSS size before/after : " + details.stats.originalSize + "/" + details.stats.minifiedSize + "o [" + Math.round(details.stats.efficiency * 100) + "%]");
})
```

How I can add this file/path in my console.log ?

Thanks for in advance your help

**Comment by scniro** [comment]
Hey!

Thanks for bringing this to my attention. So if you checked out the [clean-css](https://github.com/jakubpawlowicz/clean-css) repo, I'm simply returning the object and I don't see file path as one of them. However, I totally get why this would be helpful and I fully agree.

I'll work on this and keep you posted with progress.

**Comment by Ducatel** [comment]
Nice, thanks ;)

**Comment by scniro** [comment]
All set, you can now log out `name` and `path` on the `details` object 9747c288c1206638f92a0bfb330c9ec5cd22841c

published [2.0.3 to npm](https://www.npmjs.com/package/gulp-clean-css). Please let me know if this now works as expected for you.

**Comment by Ducatel** [comment]
It's working perfectly ;)
Thanks
